Protos/V1/agent_common.proto

```diff
@@ -70,13 +72,145 @@ message Result {
     InitKeyedDataStream init = 1;
     DataChunk data = 2;
   }
-  string communication_token = 3;
+  string communication_token = 3; /** Communication token received by the worker during task processing */
 }
 
 message ResultReply {
-  string communication_token = 3;
+  string communication_token = 3; /** Communication token received by the worker during task processing */
   oneof type {
     Empty Ok = 1;
     string Error = 2;
   }
 }
+
+/*
+* Request for creating results without data
+*/
+message CreateResultsMetaDataRequest {
+  /**
+  * A result to create.
+  */
+  message ResultCreate {
+    string name = 1; /** The result name. Given by the client. */
+  }
+  repeated ResultCreate results = 1; /** The list of results to create. */
+  string session_id = 2; /** The session in which create results. */
```

Comment:
We needed this information when the client was able to create the result_id. I think we can have them for consistency.

Reply:
I'm not convinced about the consistency as I think the communication_token can replace the session id. In fact, it might be possible to factorize the client proto with the agent proto if we say that you have a session id for the client and a communication token for the agent.

But all in all, I don't really care.

Reply:
We need to take that into account for the V4